src/model.py

```diff
@@ -151,6 +153,11 @@ def init_engine(self):
             AsyncEngineArgs(**self.vllm_engine_config)
         )
 
+        # Create vLLM custom Metrics
+        labels = {"model": "vllm_metrics", "version": "1"}
+        logger = VllmStatLogger(vllm_labels=labels)
```

Comment:
I think the `vllm_labels` arg name is wrong

Reply:
Fixed